gsmphys/sfc_cice.f

```diff
@@ -104,6 +105,11 @@ subroutine sfc_cice                                               &
           tem     = 1.0 / rho(i)
           hflx(i) = dtsfc(i) * tem * cpinv
           evap(i) = dqsfc(i) * tem * hvapi
+          if (evap(i) .lt. -maxevap(i)) then
+            chh(i) = -maxevap(i)/evap(i)*chh(i)
+            hflx(i) = -maxevap(i)/evap(i)*hflx(i)
```

Comment:
Checking order of operations is supposed to be `(maxevap/evap) * _var_` and you want to negate it.